calyx/src/passes/externalize.rs

```diff
@@ -74,20 +70,17 @@ fn has_external_attribute(cr: &RRC<ir::Cell>) -> bool {
 }
 
 impl Visitor for Externalize {
-    fn require_postorder() -> bool {
-        true
-    }
-
     fn start(
         &mut self,
         comp: &mut ir::Component,
         _ctx: &LibrarySignatures,
         _comps: &[ir::Component],
     ) -> VisResult {
+        let mut _port_names = HashMap::new();
         dump_ports::dump_ports_to_signature(
             comp,
             has_external_attribute,
-            &mut self.port_names,
+            &mut _port_names,
```

Comment:
Why does this name start with an `_` if it is used? Usually we start names with `_` to signal they are unused